### PR TITLE
adding README and updating contributing instructions

### PR DESCRIPTION
**Pull request by alecgunny**
I think this is just missing a description of the evaluation process but don't know how much detail is necessary to get into at the root level, so maybe leave this for the future as we work out the documentation for the projects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# BBHNet
+Detecting binary blackhole mergers from gravitational wave strain timeseries data using neural networks, with an emphasis
```
Comment (wbenoit26) [inline README.md:2]: emphasis on*

Comment (wbenoit26) [reply README.md:2]: Also, throughout, black hole should be two words

**Review by wbenoit26 — COMMENTED** [review]
I found only a couple of typos, so I'm good with this once the tests are passing

**Comment by alecgunny** [comment]
@wbenoit26 for some reason the pre-commit hook isn't showing as finished here, even though it shows as passing under the Actions tab. The unit-tests tab shows a failure there, but that's only because the lib matrix is empty because there's no library changes to test. I would suggest therefore that this is ready to go from a testing standpoint, even if we have to technically bypass the branch protections to merge it.

**Review by wbenoit26 — APPROVED** [review]
Sounds good to me

**Pull request merged**